apps/svelte.dev/content/tutorial/+assets/svelte.config.js

```diff
@@ -0,0 +1,10 @@
+import adapter from '@sveltejs/adapter-auto';
+
```

Comment:
we can't import something that doesn't exist in node_modules, and it's probably not worth installing it (which adds bloat to the webcontainer) just for the sake of this file

```suggestion
```